static/_redirects

```diff
@@ -100,6 +100,8 @@
 /guides/comparing-runs /guides/runs 301
 /guides/config /guides/sweeps/define-sweep-configuration 301
 /guides/configurations /guides/track/config 301
+/guides/core/tables /guides/models/tables 301
+/guides/tables /guides/models/tables 301
```

Comment:
Do we need to add each page that is moved? e.g. /guides/models/registry/lineage.md -> /guides/core/registry/lineage.md ?

Reply:
No but we can use wildcards: https://harrycresswell.com/writing/hugo-netlify-redirects/#understanding-the-_redirects-file-syntax I will update them accordingly. I thought it defaulted to globbing the pages within a path.

Reply:
To prove that splats work, go to https://docs-1245.docodile.pages.dev/guides/tables/tables-walkthrough/ and see that the `/models/` portion gets added to the URL upon redirect.